src/mgmt/srv_drpc.c

```diff
@@ -990,55 +1046,65 @@ add_props_to_resp(daos_prop_t *prop, Mgmt__PoolGetPropResp *resp)
 {
 	Mgmt__PoolProperty	**resp_props;
 	struct daos_prop_entry	*entry;
-	int			 i, rc = 0;
+	int			 i, rc = 0, valid_prop_nr = 0, j = 0;
```

Comment:
The DAOS style guide suggests putting each variable declaration on its own line. I know we don't always stick to that, but this line is getting long and includes, assignments, too. I think it would be best to break it up into individual lines.